dev-requirements.txt

```diff
@@ -6,3 +6,5 @@ wheel
 pytest-cov
 pre-commit
 requests
+mercurial
+hg-evolve
```

Comment:
These should be in `requirements.txt` not in `dev-requirements.txt` I think as they are needed by users when they want to run repo2docker

Reply:
It seems that there is no `requirements.txt`. The new requirements are in the Pipfile.

Reply:
Ah, sorry, we use `setup.py` directly, so the dependencies should be listed there.

Reply:
Requirements added in setup.py!